portfolio/src/main/webapp/script.js

```diff
@@ -27,16 +27,23 @@ function addRandomFact() {
   factContainer.innerText = fact;
 }
 
-async function getMessage() {
-    // fetch('/data').then(response => response.json()).then((messages) => {
+function getMessage() {
+    fetch('/data')
+    .then(response => response.json()).then((messages) => {
 
-    //     const messageList = document.getElementById('message-container');
-    //     messageList.innerHTML = ''
-    //     messages.
-    // }    
-    const response = await fetch('/data')
-    const quote = await response.text()
-    document.getElementById('message-container').innerText = quote;
+        const messageList = document.getElementById('message-container');
```

Comment:
My understanding = document.getElementById returns single (not multiple) element.
Why do you think input message has properties message1, ... ? This assumption should be documented.

Reply:
Disregard note about document.getElementById, though not regarding messages properties still holds. If it is part of requirements then reflect that in function or code comment.

Reply:
Ok, I will!